cardinal/world.go

```diff
@@ -32,9 +32,9 @@ type World struct {
 type (
 	// EntityID represents a single entity in the World. An EntityID is tied to
 	// one or more components.
-	EntityID     = entity.ID
-	TxHash       = transaction.TxHash
-	WorldContext = ecs.WorldContext
+	EntityID        = entity.ID
+	TxHash          = transaction.TxHash
+	ECSWorldContext = ecs.WorldContext
```

Comment:
can't. This library is a wrapper around ecs.WorldContext.

I think you're shooting for a re-achitecture here, please be more explicit about the intent here. 

Given that both Tyler and Namra believe this will be re-architected I suggest we compromise here as it's going to be replaced.

Reply:
ECSWorldContext is just a type def, so any instances of ECSWorldContext can be replaced with `ecs.WorldContext` and have the same behavior. I see 6 instances of ECSWorldContext (2 each in read.go, world.go, and world_context.go). Just replace ECSWorldContext with ecs.WorldContext. 

This eliminates the ECSWorldContext from the godoc page and has no functional changes in the cardinal package.

Reply:
ok I thought you meant remove the ecs.WorldContext itself.